### PR TITLE
Feature branch

### PR DESCRIPTION
**Pull request by icebox827**
# Pull request for the Library project
- Create the myLibrary array
- Create the createBook function
- Create the displayBook function
- Create the clearInputField function
- Create the form to add book
- Create the the read button
- Create  the remove button
- Use the local storage

### CODE REVIEW

**Review by teekaytech — CHANGES_REQUESTED** [review]
## Status: Changes Required :warning: :vertical_traffic_light: 

### Hello team :wave:

__So far, you've made reasonable progress :handshake:__

_**Major highlights:**_

- Your PR title and summary is descriptive :+1: 
- Your commit messages are concise :ok: 

__However, some changes are required before this project can be approved. Please check my comments below and fix them as appropriate.__

**[ REQUIRED CHANGES ]**

_**Please note that you're expected to follow the step by step instructions in building the project as highlighted on `The Odin Project` page**_

- [x] Update your Readme file. You're expected to implement all the general requirements of this project. This includes setting up a Readme file that contains all the necessary sections as you have in previous projects.
- [x] The application have a `addBookToLibrary` function. I'm sure you've implemented this already in your project. However, naming the function `addBookToLibrary` is a core requirement that must be obeyed  ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/library/book_array_requirement.png) from [Odin](https://www.theodinproject.com/courses/javascript/lessons/library)). This function should be able to take the user's input, then store the new Book objects into an array.
- [x] There should be a button named `New Book` on your page.
- [x] When the `New Book` button is clicked, it should bring up a form with the following fields: `author`, `title`, `number of pages`, and` book status`.
form

- [x] Try to keep the application logic separated from DOM manipulation tasks ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/section/dom_logic_separation_requirement.png) from Pathwright). For instance, you can extract the `book`, `storageMyLibrary`, `reloadPage` functions (etc) to a separate JS script. This will make your logic easy to test and scale in the future.

**[ OPTIONAL CHANGES ]**

- [ ] It will be nice if you can add validation rules to the form used to create a book so that users will not be able to create an empty book :blush: 

* * *
**Please make the required changes and submit another review request when you are done.**

* * *

**Taofeek Olalere
[Twitter](https://twitter.com/ola_lere) | [Github](https://github.com/teekaytech) | @taofeekolalere on Slack**
__Happy Coding :smiley:__

File: README.md
Comment (christianotieno) [inline README.md]: ## Fix Screenshot on README
- [ ] The screenshot on this `README` is currently not showing. Do you mind improving it. it should be a screenshot of a section of the page.

File: assets/Script/main.js
Comment (christianotieno) [inline assets/Script/main.js:26]: ## Fix book creation 
- [ ] As a user, I expect the form to be submitting the form values and saving my book. It is currently throwing an error at the console when a user tries to save a book. below I have shared a screenshot of the same about these specific errors. Take a look and fix this, kindly, thank you.

library

**Review by christianotieno — CHANGES_REQUESTED** [review]
## Changes Required ♻️ 
Hello @icebox827 & @gasb150 👋🏾 

#### Summary
Thank you for implementing the suggested changes issued earlier. The project is now taking shape and we are almost getting this approve. The recent changes might have introduced some bug to our code.  

##### Changes
- [ ] User is unable to create a book (highlighted below 👇🏾 )
- [ ] Improve the screenshot in the `README`.

### Happy coding 💻 
#### Christian Otieno (CR)

**Review by christianotieno — APPROVED** [review]
### Project Approved 🍾 
Hello @icebox827 & @gasb150 👋🏾 
##
These changes have now met all the requirements needed for approval. 👏🏾 
Congratulations on the hard work done with the changes and overall, on this project.  👍🏾 
I will go ahead and mark this as complete. Good luck with the next tasks 🥇 
##
### Happy Coding 💻 
#### Christian Otieno (CR)
**| [Twitter](https://twitter.com/iamchrisotieno) | [Github](https://github.com/christianotieno) | [LinkedIn](https://www.linkedin.com/in/christianotieno/) |**

**Pull request merged**
